backend/wmg/pipeline/summary_cubes/cell_count.py

```diff
@@ -92,6 +94,11 @@ def create_formatted_citation(collection):
     return dataset_dict
 
 
+def _get_ancestors_and_agg(x):
```

Comment:
this is used to aggregate ancestors of a cell type into one element (delimited by ';').